style.css

```diff
@@ -150,17 +143,167 @@ nav.active .menu-items li {
+}
+
+/* pop up section */
+
 .popup-bg.show {
   position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
+  top: 50%;
+  left: 50%;
+  padding: 1rem;
+  transform: translate(-50%, -50%);
   backdrop-filter: blur(8px);
   background-color: #c1c7d0;
-  height: 700px;
+  width: 100%;
+  min-height: 100%;
   mix-blend-mode: multiply;
 }
 
+.pop-up-section {
+  display: none;
+}
+
+.pop-up-section.show {
+  position: fixed;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  gap: 1rem;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  width: 90%;
+  height: 95%;
+  overflow: scroll;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 1rem;
+}
+
+.pop-up-section.show .project-description-container-popup {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.techs-and-buttons {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 2rem;
+}
+
+.techs-and-buttons .live-and-source-btn {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1rem;
+  padding: 0.5rem;
+}
+
+.techs-and-buttons .live-and-source-btn a {
+  padding: 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 6px;
+  color: #396df2;
+  letter-spacing: 0.03em;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  text-decoration: none;
+  transition: background-color 1s ease;
+}
+
+.techs-and-buttons .live-and-source-btn a .live-icon {
+  width: 1.2rem;
+}
+
+.techs-and-buttons .live-and-source-btn a .github-icon {
+  width: 1rem;
+}
+
+.techs-and-buttons .live-and-source-btn a:enabled {
+  background: #fff;
+  border: 1px solid #4d5bd9;
+  border-radius: 8px;
+}
+
+.techs-and-buttons .live-and-source-btn a:hover {
+  background-color: #2d40e6;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.techs-and-buttons .live-and-source-btn a:active {
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.techs-and-buttons .live-and-source-btn a:disabled {
+  color: #5e6c84;
+  background: #e5e5e5;
+}
+
+.toggle-to:enabled {
+  background: #fff;
+  border: 1px solid #4d5bd9;
+  border-radius: 8px;
+}
+
+.toggle-to:hover {
+  background-color: #2d40e6;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.toggle-to:active {
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.toggle-to:disabled {
+  color: #5e6c84;
+  background: #e5e5e5;
+}
+
+.pop-up-section.show .mobile {
+  display: block;
+}
+
+.pop-up-section.show .desktop {
+  display: none;
+}
+
+.pop-up-section.show .toggle-to {
+  position: absolute;
+  top: 1rem;
+  right: 2rem;
+  font-family: 'Poppins', sans-serif;
+  text-decoration: none;
+  border-radius: 16px;
+  color: #396df2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  cursor: pointer;
+  transition: background-color 1s ease;
+}
```

Comment:
- [] Kindly make sure to stick to the design by letting all your popup display and scrolling the whole overlay instead :recycle: 

HINT: for this you can set overflow hidden for the popup, after making sure it's heignt is auto, then set overflow-y:scroll for the overlay section :recycle: 

Your design:

![Screenshot from 2023-02-04 21-20-15](https://user-images.githubusercontent.com/97834160/216786684-824fad8f-b77c-4eea-a429-ab4e3f335e07.png)

Figma design:

![Screenshot from 2023-02-04 21-30-14](https://user-images.githubusercontent.com/97834160/216786688-734f4797-9b44-4986-a7fb-37dea71f25af.png)